src/client/src/components/icons/UndockIcon.tsx

```diff
@@ -3,11 +3,14 @@ interface Props {
   height: number
 }
 
-const UndockIcon: React.FC<Props> = props => {
-  const { width, height } = props
-
+const UndockIcon: React.FC<Props> = ({ width, height }) => {
```

Comment:
needs the default width/height treatment too, I think - this is another one that only appears in the styleguide
(might have a word with Andres and get UX to periodically review that)